docs/features.md

```diff
@@ -0,0 +1,297 @@
+---
+title: Features
```

Comment:
Maybe better use feature flags? We have more features probably in busola than in this doc.

Reply:
Maybe both? "Features and feature flags"?